packages/tools/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -28,7 +28,10 @@ for (const subCommand of realCommand.split(/&&/g)) {
 
 function getRealCommand(cmd, toolsRoot) {
     const eslintCommand = "eslint --config .eslintrc.js --ext .jsx,.js,.ts,.tsx src";
-    const prettierCommand = 'prettier --config prettier.config.js "{src,tests}/**/*.{js,jsx,ts,tsx}"';
+    const prettierCommand = `prettier --config "${join(
+        __dirname,
+        "../../../../prettier.config.js"
```

Comment:
Do we need this kind of horrible path? We removed a while ago from the npm tasks..

Reply:
I have a feeling if we use this it will fail in generated widgets using the piw-tools outside the mono repo.

Reply:
I think so. We couldn't come up with a better solution as `process.cwd()` will be dynamic (the folder of the widget it runs in). 
`__dirname` will always translate to the same folder (`packages/tools/pluggable-widgets-tools/bin/mx-scripts.js`).

Reply:
This needs to be tested over a widget generated from the generator to see if this doesn't have any further impact.

Reply:
The `toolsRoot` is also available but it will only avoid one `../`